### PR TITLE
Added redcarpet gem and markdown for shares

### PR DESCRIPTION
**Pull request by kristakarpan**

### CODE REVIEW

File: app/views/shares/index.html.erb [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <div class="handle"><%= current_user.name %></div>
     <%= link_to share.title, share_path(share) %>
     <p>
-      <%= linkify_hashtags(share.body) %>
+      <%= linkify_hashtags.markdown(share.body) %>
```
Comment (kenmazaika) [inline app/views/shares/index.html.erb]: Does this actually work?  I would expect the code to look more like:

```
linkify_hashtags(markdown(share.body))
```

Unless there's some way the hashtag gem and redcarpet interact that I don't understand.

**Comment by kenmazaika** [comment]
Looks good, I had one question though.

**Comment by kristakarpan** [comment]
No, you're right. Went ahead and adjusted it!

**Comment by kenmazaika** [comment]
Perfect.  Is this ready to merge?  If so I'll press the buttons when the tests finish.

**Comment by kristakarpan** [comment]
Should be ready to merge!

**Comment by kenmazaika** [comment]
Nice!

**Pull request merged**